pkg/apis/apps/v1beta2/conversion.go

```diff
@@ -361,3 +368,89 @@ func Convert_extensions_RollingUpdateDeployment_To_v1beta2_RollingUpdateDeployme
 	}
 	return nil
 }
+
+func Convert_extensions_DaemonSet_To_v1beta2_DaemonSet(in *extensions.DaemonSet, out *appsv1beta2.DaemonSet, s conversion.Scope) error {
+	out.ObjectMeta = in.ObjectMeta
+	if out.Annotations == nil {
+		out.Annotations = make(map[string]string)
+	}
+	out.Annotations[appsv1beta2.DeprecatedTemplateGeneration] = strconv.FormatInt(in.Spec.TemplateGeneration, 10)
+	if err := Convert_extensions_DaemonSetSpec_To_v1beta2_DaemonSetSpec(&in.Spec, &out.Spec, s); err != nil {
+		return err
+	}
+	if err := s.Convert(&in.Status, &out.Status, 0); err != nil {
```

Comment:
It looks like there is no generated conversions functions for the Status, could you check why?

You perhaps need to add a new comment `// +k8s:conversion-gen=k8s.io/kubernetes/pkg/apis/extensions` in `k8s.io/kubernetes/pkg/apis/v1beta2/doc.go`